### DISCUSSION TITLE
Purge modified files from CDN Cache

### DISCUSSION THREAD

**Discussion by laukstein** [BetaHuhn/do-spaces-action#435] [Ideas]
[do-spaces-action](https://github.com/BetaHuhn/do-spaces-action) successfully uploads files but doesn't purge it from DigitalOcean CDN Cache.

Can do it like
```
curl -X DELETE -H "Content-Type: application/json" \
    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
    -d '{"files": ["assets/img/hero.png","assets/css/*"]}}' \
    "https://api.digitalocean.com/v2/cdn/endpoints/<CDN_ENDPOINT_ID>/cache" 
```

API doc in https://docs.digitalocean.com/reference/api/api-reference/#operation/purge_cdn_cache
Example https://www.digitalocean.com/community/questions/api-to-purge-file-cache-from-spaces-cdn-after-file-update?comment=153381

**Comment by BetaHuhn** [comment]
Good point, we should definitely add this.

Feel free to create a PR for this (maybe use `node-fetch` instead of calling curl) and I will review and merge it.

**Comment by dannymout** [comment]
opened #1052 to add this feature
